examples/MuttsVersionedTrees/src/MuttsVersionedTrees.jl

```diff
@@ -83,15 +83,16 @@ function AbstractTrees.children(v::VTreeNode)
     end
 end
 function AbstractTrees.printnode(io::IO, v::VTreeNode{T}) where T
-    mutable_emoji = ismutable(v) ? "✔︎" : "🔒"
+    mutable_emoji = is_mutts_mutable(v) ? "✔︎" : "🔒"
     print(io,"VTreeNode{$T}($(v.value)) $mutable_emoji")
 end
 
 Base.show(io::IO, v::VTree) = AbstractTrees.print_tree(io, v)
 
 # ----- Copy needed for Mutts.branch!() ---------------
-Base.copy(v::EmptyVTree{T}) where T = EmptyVTree{T}()
-Base.copy(v::VTreeNode{T}) where T = VTreeNode{T}(v.value, v.left, v.right)
+import Mutts.make_mutable_copy
```

Comment:
```suggestion
```
This is not needed; you can already add methods to a function just by using the fully-qualified name, and I think this is what is preferred. (Although it doesn't seem to be specified in our style guide! 😮 https://github.com/RelationalAI/RAIStyle#module-imports)

Reply:
Oh, TIL. Thanks for noticing that.